pydantic_extra_types/types/country.py

```diff
@@ -0,0 +1,1156 @@
+    'Mauritania': 'MR',
+    'Mauritius': 'MU',
+    'Mayotte': 'YT',
+    'Mexico': 'MX',
+    'Micronesia (Federated States of)': 'FM',
+    'Moldova (the Republic of)': 'MD',
+    'Monaco': 'MC',
+    'Mongolia': 'MN',
+    'Montenegro': 'ME',
+    'Montserrat': 'MS',
+    'Morocco': 'MA',
+    'Mozambique': 'MZ',
+    'Myanmar': 'MM',
+    'Namibia': 'NA',
+    'Nauru': 'NR',
+    'Nepal': 'NP',
+    'Netherlands': 'NL',
+    'New Caledonia': 'NC',
+    'New Zealand': 'NZ',
+    'Nicaragua': 'NI',
+    'Niger': 'NE',
+    'Nigeria': 'NG',
+    'Niue': 'NU',
+    'Norfolk Island': 'NF',
+    'Northern Mariana Islands': 'MP',
+    'Norway': 'NO',
+    'Oman': 'OM',
+    'Pakistan': 'PK',
+    'Palau': 'PW',
+    'Panama': 'PA',
+    'Papua New Guinea': 'PG',
+    'Paraguay': 'PY',
+    'Peru': 'PE',
+    'Philippines': 'PH',
+    'Pitcairn': 'PN',
+    'Poland': 'PL',
+    'Portugal': 'PT',
+    'Puerto Rico': 'PR',
+    'Qatar': 'QA',
+    'Réunion': 'RE',
+    'Romania': 'RO',
+    'Russian Federation': 'RU',
+    'Rwanda': 'RW',
+    'Saint Barthélemy': 'BL',
+    'Saint Helena Ascension Island Tristan da Cunha': 'SH',
+    'Saint Kitts and Nevis': 'KN',
+    'Saint Lucia': 'LC',
+    'Saint Martin (French part)': 'MF',
+    'Saint Pierre and Miquelon': 'PM',
+    'Saint Vincent and the Grenadines': 'VC',
+    'Samoa': 'WS',
+    'San Marino': 'SM',
+    'Sao Tome and Principe': 'ST',
+    'Saudi Arabia': 'SA',
+    'Senegal': 'SN',
+    'Serbia': 'RS',
+    'Seychelles': 'SC',
+    'Sierra Leone': 'SL',
+    'Singapore': 'SG',
+    'Sint Maarten (Dutch part)': 'SX',
+    'Slovakia': 'SK',
+    'Slovenia': 'SI',
+    'Solomon Islands': 'SB',
+    'Somalia': 'SO',
+    'South Africa': 'ZA',
+    'South Georgia and the South Sandwich Islands': 'GS',
+    'South Sudan': 'SS',
+    'Spain': 'ES',
+    'Sri Lanka': 'LK',
+    'Sudan': 'SD',
+    'Suriname': 'SR',
+    'Svalbard Jan Mayen': 'SJ',
+    'Sweden': 'SE',
+    'Switzerland': 'CH',
+    'Syrian Arab Republic': 'SY',
+    'Taiwan (Province of China)': 'TW',
+    'Tajikistan': 'TJ',
+    'Tanzania, the United Republic of': 'TZ',
+    'Thailand': 'TH',
+    'Timor-Leste': 'TL',
+    'Togo': 'TG',
+    'Tokelau': 'TK',
+    'Tonga': 'TO',
+    'Trinidad and Tobago': 'TT',
+    'Tunisia': 'TN',
+    'Türkiye [ab]': 'TR',
+    'Turkmenistan': 'TM',
+    'Turks and Caicos Islands': 'TC',
+    'Tuvalu': 'TV',
+    'Uganda': 'UG',
+    'Ukraine': 'UA',
+    'United Arab Emirates': 'AE',
+    'United Kingdom of Great Britain and Northern Ireland': 'GB',
+    'United States Minor Outlying Islands': 'UM',
+    'United States of America': 'US',
+    'Uruguay': 'UY',
+    'Uzbekistan': 'UZ',
+    'Vanuatu': 'VU',
+    'Venezuela (Bolivarian Republic of)': 'VE',
+    'Viet Nam': 'VN',
+    'Virgin Islands (British)': 'VG',
+    'Virgin Islands (U.S.)': 'VI',
+    'Wallis and Futuna': 'WF',
+    'Western Sahara': 'EH',
+    'Yemen': 'YE',
+    'Zambia': 'ZM',
+    'Zimbabwe': 'ZW',
+}
+
+
```

Comment:
I guess we can create a folder called `constant` and add this as `country.py` have all of the `dict` we have here and also in the future when we will be able to support other constants such lists and dicts

cc @samuelcolvin what you think about this

Reply:
Sounds like a good idea. It will improve code readability. I am waiting for Samuel's response for further changes.